master/pkg/checkpoints/local/local_checkpoints.go

```diff
@@ -115,12 +115,13 @@ func NewLocalDownloader(aw archive.ArchiveWriter, prefix string) (*LocalDownload
 	if !strings.HasPrefix(prefix, "/") {
 		prefix = "/" + prefix
 	}
+	prefix = filepath.Clean(prefix)
 	if !strings.HasSuffix(prefix, "/") {
 		prefix += "/"
 	}
 	return &LocalDownloader{
 		aw:     aw,
-		prefix: filepath.Clean(prefix),
```

Comment:
why do we need ```filepath.Clean``` in the first place?

Reply:
the original instance of `filepath.Clean` in `archivePath` was added because the go sec linter desired some validation for open, altho one could argue we could ignore the linter there. it does seem helpful to cleanup the input here as we don't need to be as concerned with the internal implementation of `filepath.Walk` and what sort of path names might be returned when we trim the prefix during listing.